portfolio/src/main/webapp/style.css

```diff
@@ -3,6 +3,52 @@
   margin-right: auto;
   width: 650px;
 }
+body {
+  background-image: url("images/Greywatercolor.jpg");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+h1{
+    text-align: center;
+    font-family: 'Prata', serif ;
+}
+.nav { /*entire nav bar*/
```

Comment:
I like that you preferred css classes that are shared amongst your components instead of having many ID selectors to do custom styling on each element. Well done, Mannie!